src/Options/index.js

```diff
@@ -94,6 +94,12 @@ export interface ParseServerOptions {
   databaseOptions: ?DatabaseOptions;
   /* Adapter module for the database */
   databaseAdapter: ?Adapter<StorageAdapter>;
+  /* Disable case insensitivity (collation) on queries and indexes, needed if the you use mongodb serverless
+  :ENV: PARSE_SERVER_DISABLE_DEFAULT_CASE_INSENSITIVITY */
+  disableCaseInsensitivity: ?boolean;
+  /* Force username and email to lowercase on create/update/login/signup. On queries client needs to ensure to send lowercase email/username
+  :ENV: PARSE_SERVER_FORCE_EMAIL_AND_USERNAME_TO_LOWER_CASE */
+  forceEmailAndUsernameToLowerCase: ?boolean;
```

Comment:
- Instead of "force" let call it "convert to".
- Notation as if these were full sentences, this goes into the API docs.

Reply:
what about ? `transformUsernameToLowerCase` and `transformEmailToLowerCase`

Reply:
See https://github.com/parse-community/parse-server/pull/8042#discussion_r912342663

Reply:
Sounds good